docs/articles/documentation/recipes/running-tests-in-firefox-and-chrome-using-travis-ci.md

```diff
@@ -0,0 +1,100 @@
+
+## Step 1 - Install TestCafe and create tests
+
+Install TestCafe [locally](../using-testcafe/installing-testcafe.md#locally) in your project and [create tests](../getting-started/#creating-a-test).
+
+## Step 2 - Enable Travis for your project
+
+1. [Sign in to Travis CI](https://travis-ci.org/auth) with your GitHub account. Travis CI will synchronize your repositories from GitHub. You can see them on your [profile page](https://travis-ci.org/profile).
+2. Enable Travis CI for a repository you want to build by flicking the switch on.
+
+     ![Enable Travis for a repository](../../images/travis-step-2-2.png)
+
+     By default, Travic CI runs builds on pushes and pull requests. You can manage this behavior in your repository's settings.
+
+     ![Enable builds](../../images/travis-step-2-4.png)
+
+3. Add a `.travis.yml` configuration file to the root of your project. This file should contain parameters and commands that instruct Travis CI how to execute your builds. For more information, see [Customizing the Build](https://docs.travis-ci.com/user/customizing-the-build).
+
+     For Node.js projects, the `.travis.yml` file can have the following content.
+
+     ```yaml
+     language: node_js
+     node_js: "stable"
+     ```
+
+     Commit and push this file to your repository.
+
+## Step 3 - Configure Travis to use Xvfb
+
+Travis CI uses Ubuntu Server virtual machines, that do not have regular graphical environment like GNOME or KDE installed, So you have to setup and use [Xvfb](https://www.x.org/archive/X11R7.6/doc/man/man1/Xvfb.1.xhtml) to run browsers headlessly. 
+
+You should add the following sections to your `.travis.yml` to start `Xvfb`:
+
+```
+dist: trusty
+sudo: required
+
+addons:
+  firefox: latest
+  apt:
+    sources:
+     - google-chrome
+    packages:
+     - google-chrome-stable
+     
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3
+```
+
+You can find more information about Travis and Xvfb in [this article](https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI).
+
+## Step 4 - Add the `test` script to package.json
+
+To test a project, Travis runs test scripts. For Node.js projects, the default test script is `npm test`.
+To tell npm how to run your tests, you need to add the `test` script to the project's package.json file. The script should contain a `testcafe` command to run tests on a Sauce Labs browser.
+
+```text
+"scripts": {
+    "test":  "testcafe chrome,firefox tests/index-test.js"
+}
+```
+
+For more information on how to configure a test run using a `testcafe` command, see [Command Line Interface](../using-testcafe/command-line-interface.md).
+
+**Important:** If you are going to run tests for a website that is not deployed, the `test` script should also include commands to run the site. 
```

Comment:
I don't like it in the form of `if you need, then you should`. It should smthg like: "if you need to run tested app server as well, you can use TestCafe `--app-command` flag: TestCafe will start server before testing and terminate it once the testing is done". Also, it shouldn't be marked as `Important`. It's just a note